layouts/partials/favicon.html

```diff
@@ -0,0 +1,9 @@
+<link rel="apple-touch-icon" sizes="180x180" href="/images/favicon/apple-touch-icon.png">
+<link rel="icon" type="image/png" sizes="32x32" href="/images/favicon/favicon-32x32.png">
+<link rel="icon" type="image/png" sizes="16x16" href="/images/favicon/favicon-16x16.png">
+<link rel="manifest" href="/images/favicon/site.webmanifest">
+<link rel="mask-icon" href="/images/favicon/safari-pinned-tab.svg" color="#5bbad5">
+<link rel="shortcut icon" href="/images/favicon/favicon.ico">
+<meta name="msapplication-TileColor" content="#da532c">
```

Comment:
This is an orangey color, I don't think it's going to look well with the icon?
<img width="563" alt="Screenshot 2022-01-18 at 12 48 38" src="https://user-images.githubusercontent.com/1009343/149970746-b824c90e-8018-449e-80b6-a839dd424e32.png">

Reply:
Hello @eternoendless.

I think it's a job for the product team to choose the color :)

Reply:
Let's not wait for the product team on such a little thing I think so, I would use white color probably

Reply:
![Capture d’écran de 2022-01-19 09-11-39](https://user-images.githubusercontent.com/5064590/150090265-d9bf1b5c-bd8e-4d62-9d56-2982b14d068e.png)

With the actual color